src/core/client/admin/routes/Configure/ConfigureRoute.tsx

```diff
@@ -28,7 +29,8 @@ class ConfigureRoute extends React.Component<Props, State> {
     data: Parameters<Props["updateSettings"]>[0]["settings"],
     form: FormApi
   ) => {
-    await this.props.updateSettings({ settings: data });
+    const payload = omit(data, "announcement");
```

Comment:
I'm a bit worried about this, let's have a chat tomorrow.

Reply:
we had a chat about this in #coral-tech-talk a few days ago and couldn't come up with a good solution that didn't involve re-architecting the whole configuration section. Love to hear your suggestions on this!